docs/pages/bdd-references/given.md

```diff
@@ -101,6 +101,15 @@ I resource that supports
 |:---:|:----------|:-|
 | [something](#){: .p-1 .text-green-200 .fw-700}  | [something](#){: .p-1 .text-green-200 .fw-700} | any property within Terraform resource/provider/etc | `tags` `access_key` `ingress` `"something with spaces"` `kms_key_id` |
 
+NOTE: Some properties will act different than others based on if they are a Terraform default property for a given resource. 
```

Comment:
This is a very good point for extra explanation. Can we also mention that these default values are coming from the providers ?

Reply:
added a bit more detail in the other comment. I'll reword that comment to be a bit more descriptive.

Reply:
Thanks 🎉 Looking forward.

Reply:
Sorry just saw that change :) Resolving this one.